lib/Client/Transports/TransportHTTPJSON.php

```diff
@@ -10,6 +10,8 @@ class TransportHTTPJSON {
     protected $_host = '';
     protected $_port = 0;
     protected $_verbose = 0;
+    protected $_timeout = null;
```

Comment:
Using `null` here would override the default param but we want to use the default value. According to `fsockopen` [docs](https://www.php.net/manual/en/function.fsockopen.php), the default is `ini_get("default_socket_timeout")`. I think we should use that instead.

```suggestion
    protected $_timeout = ini_get("default_socket_timeout");
```

Reply:
Good catch. I will push the change.